ui/admin/app/templates/scopes/scope/host-catalogs/host-catalog/hosts/index.hbs

```diff
@@ -13,12 +13,14 @@
       {{#each @model as |host|}}
         <body.row as |row|>
           <row.headerCell>
-            <LinkTo
-              @route='scopes.scope.host-catalogs.host-catalog.hosts.host'
-              @model={{host.id}}
-            >
-              {{host.displayName}}
-            </LinkTo>
+            {{#if (can 'read model' host)}}
+              <LinkTo
+                @route='scopes.scope.host-catalogs.host-catalog.hosts.host'
+                @model={{host.id}}
+              >
+                {{host.displayName}}
+              </LinkTo>
+            {{/if}}
```

Comment:
Does this need an else clause?

Reply:
oops, it does! will update this and the other two PRs